### PR TITLE
Feature Add New York Times Clone 

### PR DESCRIPTION
**Pull request by Bluette1**
# Positioning and Floating elements: New York Times Clone 
- This is a clone of the [original New York times page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)
- [Live Demo Link](https://raw.githack.com/Temmarie/NewYorkTimes-clone/NYT-clone/index.html)
- We used CSS Floats and Positions to complete this [project](https://www.theodinproject.com/courses/html5-and-css3/lessons/css3-basics)

### CODE REVIEW

**Comment by Temmarie** [comment]
## Submission for project review
In this project, we used CSS and Html to clone the New York Times page and now submitting for review.

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Bluette1 and @Temmarie 
### You have done an amazing job so far, however, there are a few issues to solve.
- Write a more descriptive pull request [check this guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- Validate your html code with this [tool](https://validator.w3.org/#validate_by_upload)
vaidate
- The sidebar is missing
sidebar
- The name and the social media icons are too far apart, check the [original webite](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)
to wide
- Avoid inline css styling, put every styling in their file
- Use html5 semantics ie use `article, main, nav, section, footer, aside` most of the time, make changes where necessary
- Avoid too much html nesting, make it simple

### Make these changes and resubmit for another review. I wish you all the best in the next submission.
- Happy coding!
- For help: contact me on @rOluochKe

**Comment by Temmarie** [comment]
# Positioning and Floating elements: New York Times Clone 

- Added the editor's sidebar
- Validated the Html
- Removed all the inline styles and
- Used semantic Html tags

- [Live Demo Link](https://raw.githack.com/Temmarie/NewYorkTimes-clone/NYT-clone/index.html)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+	<title>New York times</title>
+</head>
+<body>
+	<header>
+		<nav class="nav-bar">
+			<div class="left-nav-bar">
+				<i class="nav-icon fas fa-bars"><a></a></i> <i class="fas fa-search"></i> <span class="space-cosmos"><a href="https://www.nytimes.com/section/science/space">SPACE & COSMOS</a></span>
+			</div>
+			<div class="center-nav-bar">
+				<svg height="25" width="185" xmlns="http://www.w3.org/2000/svg">
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

File: index.html [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+						<a href="#"><i class="fab fa-twitter"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><i class="fas fa-envelope"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><i class="fas fa-share"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><svg class="css-ubw31m" height="25px" viewbox="0 0 16 25" width="16px">
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

File: index.html [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+					<p>“With nature, you have to be lucky,” he said. “Apparently we have been lucky.”</p>
+					<p>The results are the closely guarded distillation of three years’ worth of observations and analysis. Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week.</p>
+					<p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.</p>
+					<p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, <a href="#">knocked on his door with a bottle of Champagne</a> to tell him the news.</p>
+					<div class="video">
+            <iframe allowfullscreen height="335" src="https://www.youtube.com/embed/ZlfIVEy_YOA" width="580"></iframe>
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

File: index.html [hunk 4]
```diff
@@ -0,0 +1,612 @@
...
+                <img alt="alamos" src="https://static01.nyt.com/images/2020/01/28/science/00NEWSPY1/merlin_166359192_36d3b52f-65fa-4a73-99ca-c9560fc52f97-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale">
+                <figcaption class="editor-outer-figcaptions">
+                  <a class="photo-by-links" href=""><section class="photo-by"><h5>Los Alamos National Laboratory</h5><br /></section></a>
+                    <a href="#" class="figcap-link">Fourth Spy at Alamos Knew A-Bomb's Inner Secrets <span class="editor-time"><br /><time datetime="2008-02-14 20:00">Jan. 27</time> </span></a>
+                </figcaption>
+            </figure>
+        </div>
+        <hr class="double-line">
+        <div class="bottom-links-container">
+          <svg class="footer-logo" height="25" width="185" xmlns="http://www.w3.org/2000/svg">
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

File: index.html [hunk 5]
```diff
@@ -0,0 +1,612 @@
...
+              </ul>
+            </div>
+            <hr class="vertical-column-line">
+            <div class="floating-links">
+              <div class="column-6">
+                
+                <h3><strong>SUBSCRIBE</strong></h3>
+                   <ul class="sub-1">
+                      <li>
+                        <a class="bottom-links"><svg class="bottom-links-image" viewBox="0 0 14 13" fill="#000"><path d="M13.1,11.7H3.5V1.2h9.6V11.7zM13.1,0.4H3.5C3,0.4,2.6,0.8,2.6,1.2v2.2H0.9C0.4,3.4,0,3.8,0,4.3v5.2v1.5c0,0.8,0.8,1.5,1.8,1.5h1.7h0h7.4h2.2c0.5,0,0.9-0.4,0.9-0.9V1.2C14,0.8,13.6,0.4,13.1,0.4"></path><polygon points="10.9,3 5.2,3 5.2,3.9 11.4,3.9 11.4,3"></polygon><rect x="5.2" y="4.7" width="6.1" height="0.9"></rect><rect x="5.2" y="6.5" width="6.1" height="0.9"></rect></svg> <b class="bottom-links-headings">Home Delivery</b></a>
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

File: index.html [hunk 6]
```diff
@@ -0,0 +1,612 @@
...
...45.1,8-44,9.4-44h0.1v1.7l-1.3,1.1L9.5-40v2.4c-0.5,0.2-1.1,0.3-1.7,0.3c-2.2,0-3.6-1.3-3.6-3.5c0-0.5,0.1-1,0.2-1.5l1.1-0.5v4.9l2.2-1v-5l-3.3,1.5c0.3-1,1-1.7,1.8-2l0,0c-2.2,0.5-4.3,2.1-4.3,4.6c0,2.9,2.4,4.8,5.2,4.8C10.2-35.1,11.9-37.1,11.9-40L11.9-40z"></path><path d="M12.2-23.7c-0.2,0-0.4,0.2-0.4,0.4v0.4L0.4-19.1v2.3l3,1l-0.2,0.6c-0.3,0.8,0.1,1.8,0.9,2.1l1.7,0.7c0.2,0.1,0.4,0.1,0.6,0.1c0.6,0,1.3-0.4,1.5-1l0.4-0.9l3.5,1.2v0.4c0,0.2,0.2,0.4,0.4,0.4c0.2,0,0.4-0.2,0.4-0.4v-10.7C12.6-23.5,12.4-23.7,12.2-23.7M7.1-13.6c-0.2,0.4-0.6,0.6-1,0.4l-1.7-0.7c-0.4-0.2-0.6-0.6-0.4-1l0.3-0.7l3.3,1.1L7.1-13.6z"></path><path d="M13.1-60.3H3.5v-10.5h9.6V-60.3zM13.1-71.6H3.5c-0.5,0-0.9,0.4-0.9,0.9v2.2H0.9c-0.5,0-0.9,0.4-0.9,0.9v5.2v1.5c0,0.8,0.8,1.5,1.8,1.5h1.7h0h7.4h2.2c0.5,0,0.9-0.4,0.9-0.9v-10.5C14-71.2,13.6-71.6,13.1-71.6"></path><polygon points="10.9,-69 5.2,-69 5.2,-68.1 11.4,-68.1 11.4,-69"></polygon><rect x="5.2" y="-67.3" width="6.1" height="0.9"></rect><rect x="5.2" y="-65.5" width="6.1" height="0.9"></rect><path d="M12,6.5H6.5V12H1V6.5h5.5V1H12V6.5zM12,0H1C0.4,0,0,0.5,0,1v11c0,0.6,0.4,1,1,1h11c0.5,0,1-0.4,1-1V1C13,0.5,12.5,0,12,0"></path></svg> <b class="bottom-links-headings">Crossword</b></a>
```
Comment (euqueme) [inline index.html]: - [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Bluette1 and @Temmarie :balloon: :lollipop: 

### First of all :clap: :clap: :clap:  claps to you for this amazing project. thank you for doing all the changes requested by the previous reviewer, your project looks almost like the original :art: lets work on  these requirements:

## More in space and Astronomy

- [ ] the title of this section should be bold just like the original

## Footer

- [ ] footer has to stick to the bottom of the page like the original

## Section Specific

- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload

- [ ] remove all inline styling from your HTML file: if an element needs inline styling change it to an element that doesn't need it (let's challenge ourselves). I have left comments where inline styling should be removed.

## Optional

[OPTIONAL] update your Live Demo, because it doesn't have all the changes you previously made

### That's it, Team! :raised_hands: 

### it has been a blast :guitar:  to review this project, please :pray: do the changes :memo: and submit back for a code review when you are done :rocket: 

any questions? reach to me in slack ** @ Maru **

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by Temmarie** [comment]
# Positioning and Floating elements: New York Times Clone
- Made the title of the recommended articles bold
- Made the footer sticky
- Removed all the inline styles
- Validated the Html and had errors as a result of removing the inline styles from the SVGs. I reported the issue to the TSE ( Maria Eugenia Queme P. ) who reviewed the project and she said to let it be since it was only on the SVGs.
- [Live Demo Link](https://raw.githack.com/Temmarie/NewYorkTimes-clone/NYT-clone/index.html)

File: index.html (hunk 6 above)
Comment (Temmarie) [reply index.html]: Thank you for the review, we have fixed the issues you pointed out, as well as the SVG inline styling we spoke about yesterday.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### You have done a great job. Congratulations :clap: :boom: I approve of your project. 
### You can now merge and submit a project completion form from your dashboard.
- Happy coding!

**Comment by Bluette1** [comment]
Thank you very much @rOluochKe!

**Pull request merged**